core/workspace_svg.js

```diff
@@ -141,16 +141,16 @@ Blockly.WorkspaceSvg = function(options,
     this.registerToolboxCategoryCallback(Blockly.PROCEDURE_CATEGORY_NAME,
         Blockly.Procedures.flyoutCategory);
   }
-
-  /**
-   * The block renderer used for rendering blocks on this workspace.
-   * @type {!Blockly.blockRendering.Renderer}
-   * @private
-   */
-  this.renderer_ = Blockly.blockRendering.init(this.options.renderer || 'geras');
 };
 Blockly.utils.object.inherits(Blockly.WorkspaceSvg, Blockly.Workspace);
 
+/**
+ * The block renderer used for rendering blocks on this workspace.
+ * @type {!Blockly.blockRendering.Renderer}
+ * @private
+ */
+Blockly.WorkspaceSvg.prototype.renderer_ = undefined;
```

Comment:
Moving forward we are no longer setting properties on the prototype to match the Google JS Style guide, and it's more performance with today's JS engines.

Reply:
No longer needed due to removing lazy loading.